code/modules/mob/living/simple_mob/subtypes/mechanical/hivebot/support.dm

```diff
@@ -0,0 +1,87 @@
+// These hivebots help their team in various ways, and can be very powerful with allies, but are otherwise very weak when alone.
+
+/mob/living/simple_mob/mechanical/hivebot/support
+	icon_state = "white"
+	icon_living = "white"
+	attacktext = list("prodded")
+	movement_cooldown = 5
+	melee_damage_lower = 2
+	melee_damage_upper = 2
+
+
+// This hivebot supplies a general buff to nearby hivebots that improve their performance.
+// Note that the commander itself does not receive the buff.
+/mob/living/simple_mob/mechanical/hivebot/support/commander
+	name = "commander hivebot"
+	desc = "A robot that appears to be directing the others."
+	maxHealth = 5 LASERS_TO_KILL // 150 health
+	health = 5 LASERS_TO_KILL
+	player_msg = "You <b>increase the performance of other hivebots near you</b> passively.<br>\
+	You are otherwise very weak offensively."
+
+/mob/living/simple_mob/mechanical/hivebot/support/commander/handle_special()
+	for(var/mob/living/L in range(4, src))
```

Comment:
I would rather avoid using ``orange()`` since by design it excludes the tile of the source, which saves a ``L == src`` check but makes it so things sharing the same tile will also always be excluded. Sharing tiles is rare but it can happen in certain situations, like spiders leaping, teleportation, someone lying down, etc, and I'd rather avoid those edge cases by using the normal ``range()`` function.